### PR TITLE
Add internal tags, remove Message from Call

### PR DESCRIPTION
**Pull request by michaelbausor**
WIP PR to add internal tags and remove references in Call to protobuf Message class, PTAL.

### CODE REVIEW

File: src/Call.php [hunk 1]
```diff
@@ -53,14 +51,14 @@ class Call
     /**
      * @param string $method
      * @param string $decodeType
-     * @param Message $message
+     * @param mixed $message
```
Comment (dwsupplee) [inline src/Call.php]: WDYT  about `Message|mixed`? Using [this](https://github.com/googleapis/gax-php/blob/master/src/ServerStream.php#L88) as precedent.

**Comment by michaelbausor** [comment]
I don't think we can do that with GapicClientTrait. It is used in generated code. If we change it in a breaking way, then anyone who upgrades their gax version but doesn't upgrade their generated code would be broken. But if we want to change it in a non-breaking way, then we are fine to do that even if it isn't marked internal. Does that make sense?

**Comment by dwsupplee** [comment]
Yeah, it does - good call.

File: src/Call.php (hunk 1 above)
Comment (michaelbausor) [reply src/Call.php]: Agreed, done.

**Comment by michaelbausor** [comment]
Ready for review, PTAL

**Pull request merged**